package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "leaflet.mouseposition",
+  "version": "0.0.1",
+  "description": "Leaflet.MousePosition is a simple mouse position control that you can drop into your leaflet map. It displays geographic coordinates of the mouse pointer, as it is moved about the map.",
+  "main": "src/L.Control.MousePosition.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lukecampbell/Leaflet.MousePosition.git"
```

Comment:
We need to drop the luke campbell repo, cant we switch to asascience-open?

Reply:
yeah def otherwise good to merge?